lib/taskana-cdi/pom.xml

```diff
@@ -179,22 +53,25 @@
 		<dependency>
 			<groupId>com.h2database</groupId>
 			<artifactId>h2</artifactId>
-			<version>1.4.194</version>
+			<version>${version.h2}</version>
 			<scope>test</scope>
 		</dependency>
 		<!-- Brought in via WildFly Swarm `bom` -->
 		<dependency>
 			<groupId>org.wildfly.swarm</groupId>
 			<artifactId>arquillian</artifactId>
+			<version>${version.wildfly.swarm}</version>
 			<scope>test</scope>
 		</dependency>
 		<!-- Brought in via Arquillian BOM, see dependencyManagement section above -->
```

Comment:
... but there is no dependencyManagement section anymore, is there?

Reply:
true, forgot to remove it.